### DISCUSSION TITLE
package org.springframework. does not exist

### DISCUSSION THREAD

**Discussion by RaffaeleVallifuoco** [microsoft/vscode-discussions#1252] [Extension Show and Tell]
Hi, everyone. I hope this is the right section.

![Screenshot 2024-06-25 alle 16 31 30 (2)](https://github.com/microsoft/vscode-discussions/assets/159124111/621480ab-e93d-47ff-824a-27cf552611da)

I recently started programming in Java using SpringBoot extension for VScode and when I open a project, I often encounter the following error: "package org.springframework.* does not exist(compiler.err.doesnt.exist)." (asterisk is used to indicate all package content)
But if I compile and run the code, it works perfectly. I tried installing a new maven dependency (like log4j, for example) but the problem persists.
I used both the latest version of the spring boot extension and the pre-release version.
Does anyone have an idea how to resolve the error ?
Another bug is the non-automatic import of sub-pasckages classes using stable versions of spring boot, but with the pre release it works perfectly.
Thank you !

**Comment by gjsjohnmurray** [comment]
No, this isn't the right place. Please see this thread which is pinned to the top of the forum.

https://github.com/microsoft/vscode-discussions/discussions/5
